[KOGITO-7705 Updating grpc documentation (#126)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/core/configuration-properties.adoc

|`_swf_eval_temp`
|No

|`kogito.grpc.enum.includeDefault`

> |Indicates if default value of enumerations should be included in grpc response.

Reviewer: ```suggestion
|Indicates if the default value of enumerations needs to be included in gRPC response.
```

|boolean

> |false

Reviewer: ```suggestion
|`false`
```

|No

|`kogito.grpc.stream.timeout`

> |Number of seconds to wait for response from server when using streams

Reviewer: ```suggestion
|Number of seconds to wait for a response from the server when using streams.
```

|int

> |20

Reviewer: ```suggestion
|`20`
```
